4_processing_features/scripts/plate1_extract_sc_cp.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process single cell morphology features for CellProfiler readouts - `Plate 1`
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pandas as pd
+
+import extraction_utils as sc_util
+
+
+# ## Set up paths to CellProfiler directory and outputs
+
+# In[2]:
+
+
+# Set file and directory constants
+cp_dir = "../CellProfiler_pipelines"
```

Comment:
Consider using `pathlib.Path` for these paths if possible (as per Way Lab conventions)

Reply:
^This comment applies to the rest of the paths in this file as well!

Reply:
Good point! I just added these to all paths but it will not work for the paths to SQLite files as it needs to be a string for the package to read the file. I have fixed all of the paths in the 4_processing_features module in commit [370b0da](https://github.com/WayScience/NF1_SchwannCell_data/pull/42/commits/370b0da035ee3d9f6fc5f7cbb65df301e46929b8).